src/components/Onboarding/PageIndicator.tsx

```diff
@@ -0,0 +1,98 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import Image from 'next/image';
+import starIcon from '@assets/star.svg';
+import { useRouter } from 'next/router';
+
+interface PageProps {
+  index: Number;
+}
+
+function PageIndicator({ index }: PageProps) {
+  const router = useRouter();
+
+  const handleSkipClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.preventDefault();
+    router.push('/start');
+  };
+
+  const handleNextClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.preventDefault();
+    if (index < 2) {
+      let nextArrow = document.getElementsByClassName('slick-next')[0] as HTMLElement;
+      nextArrow.click();
+    } else {
+      router.push('/start');
+    }
+  };
+
+  return (
+    <Styled.Root>
+      <Styled.SkipButton onClick={handleSkipClick}>건너뛰기</Styled.SkipButton>
```

Comment:
사소한 것인데, button의 기본 type은 submit입니당. submit는 input의 값을 제출하는데 쓰이는 것으로 여기서는 적절하지 않ㄴ네요~ 이런것은 type="button" 으로 바꿔주면 좋겠습니댜~!

```suggestion
      <Styled.SkipButton type="button" onClick={handleSkipClick}>건너뛰기</Styled.SkipButton>
```